src/collectors/windows.plugin/metadata.yaml

```diff
@@ -0,0 +1,265 @@
+plugin_name: windows.plugin
+modules:
+  - meta:
+      plugin_name: windows.plugin
+      module_name: PerflibProcesses
+      monitored_instance:
+        name: System statistics
```

Comment:
Do we want to have a "windows indication" here or just name it "System statistics"? If it has a windows logo, I guess we would want to make an indication about this.

Also given this monitors processes threads and context switches maybe the "System Statistics" is a bit generic, it could be "Windows Processes" or something @ilyam8 any ideas?